backend/typescript/services/implementations/campService.ts

```diff
@@ -101,6 +101,47 @@ class CampService implements ICampService {
     }
   }
 
+  async getCampById(campId: string): Promise<CampDTO> {
+    let camp: Camp | null;
+
+    try {
+      camp = await MgCamp.findById(campId);
```

Comment:
can we populate the camp sessions and form questions here so we return the objects instead of just the ids? can follow the example in getCamps https://github.com/uwblueprint/focus-on-nature/blob/de6fbd0d12001f6c633eb01b69b5a728b49389d0/backend/typescript/services/implementations/campService.ts#L34-L44